dcos/cluster.py

```diff
@@ -267,6 +260,15 @@ def get_clusters(include_linked=False):
             c = Cluster(entry)
             clusters.add(c)
 
+    # Search for linked clusters after configured clusters, this makes sure
+    # the set gives priority to Cluster over LinkedCluster.
+    if include_linked:
+        try:
+            linked_clusters = get_linked_clusters()
+            clusters.update(linked_clusters)
```

Comment:
❓ This should go in an `else` clause, only what can raise the exception should be in `try`. Optional because this is not new.

Reply:
Makes sense :+1: I'll push it tomorrow, here or otherwise into #1143